baldur/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "3.2"
```

Comment:
docker-compoe is dead; please remove from git;

you can use it if you like - just have a local copy

Reply:
Wasn't meant to be checked in, used to try the haproxy, removed